### ISSUE TITLE
Parameter tuning

### ISSUE THREAD

**Issue by leonlan** [N-Wouda/PyVRP#127]
The current HGS parameters were selected for the Euro-NeurIPS competition following a prescriptive parameter tuning procedure (see https://github.com/N-Wouda/Euro-NeurIPS-2022/issues/33).

We've already identified that these parameters are sub-optimal for the HG instances, especially on larger instances. The goal of this issue is to
- identify all parameters and their current values
- determine, for each parameter, in which values range needs to be tuned
- implement the parameter tuning script to produce parameter configs

The current main hovers somewhere around `33420` on the 1000 customer HG instances. The goal is to get about `33300` average distance.

**Comment by N-Wouda** [comment]
Oh that's probably current main after #121 got merged in :)

**Comment by leonlan** [comment]
> @leonlan which settings do you use to get to `33420` on main? What's currently set as the default?

Yes, the one with intensification. Sorry forgot to put that in the PR.

**Comment by N-Wouda** [comment]
I was a bit worried because I wasn't finding that kind of value on #120, but that's tested against the old main (before intensification). Now I get the difference, thanks!

**Comment by leonlan** [comment]
I ran https://github.com/N-Wouda/PyVRP/commit/070fb60229a1cd8a8f560c8bf93707ad7255b202 with 10 seeds:
```
Cost:  334273.3
Iters: 118266.4
```

I also ran a configuration with `repair_probability = 0.50`, `intensify_on_best = False` and `target_feasible = 0.20`. That was a bit worse:
```
Cost:  334938.0
Iters: 118043.6
```

**Comment by leonlan** [comment]
Main (seed 1) with `weight_wait_time = 0.2`, `weight_time_warp = 1.0` and `nb_granular = 40`:
```
Cost:  333681.5
Iters: 108894.0
```

Running for 10 seeds now.

**Comment by leonlan** [comment]
#141 is now merged in. I'll run a benchmark of the current main 7dd1dfe.

**Comment by leonlan** [comment]
The parameters in https://github.com/PyVRP/PyVRP/tree/main/configs work fine and the solver obtains fairly good solutions on the X and GH1000 instances. There's currently no need to tune the parameters further, so I'm closing this issue for now.
